crates/hyperswitch_connectors/src/connectors/novalnet/transformers.rs

```diff
@@ -1381,3 +1389,197 @@ pub fn get_novalnet_dispute_status(status: WebhookEventType) -> WebhookDisputeSt
 pub fn option_to_result<T>(opt: Option<T>) -> Result<T, errors::ConnectorError> {
     opt.ok_or(errors::ConnectorError::WebhookBodyDecodingFailed)
 }
+
+impl TryFrom<&SetupMandateRouterData> for NovalnetPaymentsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &SetupMandateRouterData) -> Result<Self, Self::Error> {
+        let auth = NovalnetAuthType::try_from(&item.connector_auth_type)?;
+
+        let merchant = NovalnetPaymentsRequestMerchant {
+            signature: auth.product_activation_key,
+            tariff: auth.tariff_id,
+        };
+
+        let enforce_3d = match item.auth_type {
+            enums::AuthenticationType::ThreeDs => Some(1),
+            enums::AuthenticationType::NoThreeDs => None,
+        };
+        let test_mode = match item.test_mode {
+            Some(true) => 1,
+            Some(false) | None => 0,
+        };
```

Comment:
Should we consider using constants or enums here?

Reply:
yes, sure